packages/teams-js/src/private/chat.ts

```diff
@@ -21,61 +21,53 @@ export namespace chat {
    * @hidden
    * Hide from docs
    * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
+   * Allows the user to open a chat with a single user and allows
+   * for the user to specify the message they wish to send.
    *
    * @returns Promise resolved upon completion
    */
-  export function openConversation(openConversationRequest: OpenConversationRequest): Promise<void> {
+  export function openChat(openChatRequest: OpenSingleChatRequest): Promise<void> {
     return new Promise<void>(resolve => {
       ensureInitialized(FrameContexts.content);
-      const sendPromise = sendAndHandleError('conversations.openConversation', {
-        title: openConversationRequest.title,
-        subEntityId: openConversationRequest.subEntityId,
-        conversationId: openConversationRequest.conversationId,
-        channelId: openConversationRequest.channelId,
-        entityId: openConversationRequest.entityId,
+      const sendPromise = sendAndHandleError('chat.openChat', {
+        members: openChatRequest.user,
+        message: openChatRequest.message,
       });
-      if (openConversationRequest.onStartConversation) {
-        registerHandler(
-          'startConversation',
-          (subEntityId: string, conversationId: string, channelId: string, entityId: string) =>
-            openConversationRequest.onStartConversation({
-              subEntityId,
-              conversationId,
-              channelId,
-              entityId,
-            }),
-        );
-      }
-      if (openConversationRequest.onCloseConversation) {
-        registerHandler(
-          'closeConversation',
-          (subEntityId: string, conversationId?: string, channelId?: string, entityId?: string) =>
-            openConversationRequest.onCloseConversation({
-              subEntityId,
-              conversationId,
-              channelId,
-              entityId,
-            }),
-        );
-      }
       resolve(sendPromise);
     });
   }
-
   /**
    * @hidden
    * Hide from docs
    * --------------
-   * Allows the user to close the conversation in the right pane
+   * Allows the user to create a chat with multiple users (2+) and allows
+   * for the user to specify a message and name the topic of the conversation. If
+   * only 1 user is provided into users array default back to origin openChat.
+   *
```

Comment:
We should add openChatRequest to the comments using @param

Reply:
If we make this:

@param openChatRequest {@link OpenGroupChatRequest} - a request object that contains a list of user emails as well as optional parameters for message and topic (display name for the group chat).

Then i believe the documentation will include a link to the definition of OpenGroupChatRequest

Reply:
Good idea